src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+    The annotation tsv, interval list and GATK Docker image are provided as inputs to the app,
+    but both the basecount and ploidy files are uploaded from the parent job and are available
+    in the container, so are downloaded separately.
+    '''
+    mark-section "Starting sub job to run GATK GermlineCNVCaller"
+
+    # prefixes all lines of commands written to stdout with datetime
+    PS4='\000[$(date)]\011'
+    export TZ=Europe/London
+
+    # set frequency of instance usage in logs to 10 seconds
+    kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)